packages/dds/tree/src/test/feature-libraries/flex-tree/lazyField.spec.ts

```diff
@@ -47,17 +46,23 @@ import {
 	readonlyTreeWithContent,
 	rootFieldAnchor,
 } from "./utils.js";
+import {
+	cursorFromInsertable,
+	SchemaFactory,
+	toFlexSchema,
+} from "../../../simple-tree/index.js";
+import { getFlexSchema } from "../../../simple-tree/toFlexSchema.js";
 
 const detachedField: FieldKey = brand("detached");
 const detachedFieldAnchor: FieldAnchor = { parent: undefined, fieldKey: detachedField };
 
 /**
  * Test {@link LazyField} implementation.
  */
-class TestLazyField<TTypes extends FlexAllowedTypes> extends LazyField<
-	typeof FieldKinds.optional,
-	TTypes
-> {}
+class TestLazyField<
+	TTypes extends FlexAllowedTypes,
+	TKind extends FlexFieldKind = typeof FieldKinds.optional,
```

Comment:
super nit: it just bothered me slightly that it's in a different order than LazyField, you can ignore this if you want

```suggestion
	TKind extends FlexFieldKind = typeof FieldKinds.optional,
	TTypes extends FlexAllowedTypes,
```

Reply:
your specific suggestion is invalid since you can't have parameter with a default before one without a default.

However when making this change I had incorrectly assumed some code in this file was manually specifying the parameters: I was wrong as all usages infer them, so I have removed the default and swapped the order.